src/ImageSharp/Processing/Processors/Filters/OpaqueProcessor{TPixel}.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing.Processors.Filters
+{
+    internal sealed class OpaqueProcessor<TPixel> : ImageProcessor<TPixel>
```

Comment:
Wouldn't be `MakeOpaqueProcessor` better? (Related extension to be added later: `image.MakeOpaque()`.)

Reply:
We can do that if/when we choose to expose such an extension.